static/app/views/settings/organizationAuditLog/index.tsx

```diff
@@ -104,11 +105,11 @@ function OrganizationAuditLog({location}: Props) {
     });
   };
 
+  const {isSuperuser} = ConfigStore.get('user');
```

Comment:
can we use the `isActiveSuperuser()` helper function? this ensures folks have explicitly logged in as superuser